docs/testplan/capacity_test_2TierNetwork.md

```diff
@@ -0,0 +1,31 @@
+# Test Objective
+This test aims to track switch power usage, temperature, queue watermark and health state.
+
+# Test Setup
+![Test Setup](./2TierNetwork.png)
+
+1.	The testbed consists of four IXIA traffic generators (synchronized using a time-sync metronome) and five SONiC switches, where the BT1 switch is the Device Under Test (DUT).
+2.	Each of the four BT0 switches is connected to the DUT via eight DAC cables. There are no direct connections between any two BT0 switches.
+3.	Each BT0 switch is also connected to one IXIA traffic generator via eight optical cables. Similarly, there are no direct connections between any two IXIA devices.
```

Comment:
the number of optical cables should be defined in the sonic_*_links.csv, it should be flexible and independent from the test.